indexing.go

```diff
@@ -105,33 +105,81 @@ func BulkIndex(docs []string, options Options) error {
 				return err
 			}
 
-			// Find ID in the document.
-			id, ok := docmap[options.IDField]
-			if !ok {
-				return fmt.Errorf("document has no ID field (%s): %s", options.IDField, doc)
-			}
-
+			idstring := options.IDField                            //A delimiter separates string with all the fields to be used as ID
+			id:=strings.FieldsFunc(idstring, func (r rune) bool { return r == ',' || r == ' '})			
+		//	idtemp := strings.Split(idstring, ",")
```

Comment:
Commented code is fine during development, but we should clean this up before the merge.